.github/ISSUE_TEMPLATE.md

```diff
@@ -1,19 +1,19 @@
 Thank you for reporting an issue! Please read the following before submitting:
 
-Github issues are for reporting specific bugs with Chain Core Developer Edition. If you have general questions about Chain Core or questions regarding building an application on Chain Core, please visit our developer support center: 
+Github issues are for reporting specific bugs with Chain Core Developer Edition. If you have general questions about Chain Core or questions regarding building an application on Chain Core, please visit our developer support center:
 https://support.chain.com
 
-Enterprise customers can reach out to their support contact directly. 
```

Comment:
My editor automatically stripped out these whitespaces. Is that ok? I don't think it makes any difference